include/picongpu/particles/filter/All.hpp

```diff
@@ -76,6 +76,12 @@ namespace picongpu
                 {
                     return std::string("all");
                 }
+
```

Comment:
The construction of this sentence is not easy to read. maybe
```
A filter is deterministic if the filter outcome is equal between evaluations.
If so, set this variable to true, otherwise to false.

Example: A filter were results depend on a random number generator must return false.
```

Reply:
Forgot to update the comment here?